src/parser/ultimate_guitar_parser.js

```diff
@@ -0,0 +1,105 @@
+import ChordSheetParser from './chord_sheet_parser';
+import { CHORUS, NONE, VERSE } from '../constants';
+import {
+  END_OF_CHORUS,
+  END_OF_VERSE,
+  START_OF_CHORUS,
+  START_OF_VERSE,
+} from '../chord_sheet/tag';
+
+const noop = () => {};
+const logger =
```

Comment:
I'm thinking about whether I want to have logging checked in to the codebase, although it is convenient to have when debugging. If we keep it, I'd rather move it to a separate file.

Reply:
Absolutely. Your call, let me know what to do with it.

Reply:
Can we move `logger` to `utilities.js`? Adding log calls to the parser is actually a quite good idea for better debugging 💡, so I'd propose keeping them.